### ISSUE TITLE
Implement the read-write "Payment" pane design for the "Publishers" panel

### ISSUE THREAD

**Issue by mrose17** [brave/browser-laptop#2497]
@bradleyrichter has the design token, and TBD has the implementation token.

**Comment by mrose17** [comment]
note that this is blocked on https://github.com/brave/browser-laptop/issues/2494

**Comment by bradleyrichter** [comment]
UI layout showing the initial view with autopay enabled:

![image](https://cloud.githubusercontent.com/assets/13509546/17045360/a67a2398-4f7d-11e6-8507-e6974a1e8969.png)




UI layout showing editable percentages for each publisher when auto-pay is disabled. 

![image](https://cloud.githubusercontent.com/assets/13509546/17044746/b4f35f94-4f77-11e6-9ab3-ae2ae111be69.png)




UI layout showing request to add funds to continue publisher donations:

![image](https://cloud.githubusercontent.com/assets/13509546/17045475/52564250-4f7e-11e6-88e9-3abff91d3ab9.png)



notes:
- number of entrees may vary, or become adjustable if needed.
- color of switches may change to provide better contrast for alert states
- proposed monthly budget menu choices: 2.50, 5.00, 10.00, 15.00, 20.00 USD

**Comment by BrendanEich** [comment]
$5/month minimum for 1.0 is best, based on user feedback in our home region. We'll figure out other minima for different locales for 1.1, 1.2, etc.

**Comment by diracdeltas** [comment]
closing this in favor of subissues

**Comment by diracdeltas** [comment]
@bradleyrichter it is hard to see which domain the `Include` switch is referring to, so i moved it to the far left column instead of far right
